index.html

```diff
@@ -29,6 +29,8 @@ <h2>Guides in other languages</h2>
   </div>
 </div>
 
+<button onclick="topFunction()" id="myBtn" title="Go to top"><img class="arrow-img" src="arrow.png" alt="" srcset=""></button>
```

Comment:
This location only makes it visible on the main page. Is it supposed to only be on the main page or every page?

Reply:
Yes it should be on every page. do I need to add this code manually on every page ?